src/Pages/Users/MyReservationsModal.jsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-nested-ternary */
+import React, { useState } from 'react';
+import classes from './MyReservationsModal.module.css';
+import Modal from '../../components/Modal';
+import axios from '../../config/axios';
+import GetReservationStats from '../../Api/myReservations/MyReservationsAPI';
+
+const MyReservationsModal = ((props) => {
+  const {
+    reservation, setError, setReservations, alert,
+  } = props;
+
+  const {
+    hall: {
+      capacity, cost, image, name, description,
+    }, reserve_date: reserveDate, status,
+  } = reservation;
+
+  const [close, setClose] = useState(false);
+
+  const cancelReservation = async (e) => {
+    await axios.delete('/user/reservations', { params: { reservation_id: e } }).then(() => {
+      GetReservationStats(setError, setReservations);
+      setClose(true);
+    });
+  };
```

Comment:
We already have a reservation app file, let's move it there so that we can make our component a bit clear.